main.js

```diff
@@ -0,0 +1,117 @@
+$(function() {
+    function randomString(){
+        var chars = "0123456789abcdefghiklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXTZ";
+        var str = "";
+        for (var i = 0; i < 10; i++) {
+            str += chars[Math.floor(Math.random() * chars.length)];
+        }
+        return str;
+    };
+
+    function Column(name) {
+        var self = this;
+        this.id = randomString();
+        this.name = name;
+        this.$element = createColumn();
+
+        function createColumn() {
+            var $column = $('<div>').addClass("column");
+            var $columnTitle = $("<h2>").addClass("column-title").text(self.name);
+            var $columnCardList = $("<ul>").addClass("column-card-list");
+            var $columnAddCard = $("<button>").addClass("add-card btn btn-primary").text("Add new card");
+            var $columnDelete = $("<button>").addClass("btn-delete btn btn-warning").text('x');
+            
+            $columnDelete.click(function(){
+                self.removeColumn();
+            });
+
+            $columnAddCard.click(function() {
+                self.addCard(new Card(prompt("Enter the name of the card")));
+            });
+
+            $column.append($columnTitle)
+                .append($columnAddCard)
+                .append($columnDelete)
+                .append($columnCardList);
+            
+                return $column;
+        };
+    };
+
+    Column.prototype = {
+        addCard: function(card) {
+            this.$element.children('ul').append(card.$element);
+        },
+        removeColumn: function() {
+            this.$element.remove();
+        }
+    };
+
+    function Card(description) {
+        var self = this;
+
+        this.id = randomString();
+        this.description = description;
+        this.$element = createCard();
+
+        function createCard() {
+            var $card = $("<li>").addClass("card");
+            var $cardDescription = $("<p>").addClass("card-description").text(self.description);
+            var $cardDelete = $("<button>").addClass("btn-delete btn btn-warning").text('x');
+
+            $cardDelete.click(function() {
+                self.removeCard();
+            });
+
+            $card.append($cardDescription)
+                .append($cardDelete);
+            return $card;
+        }
+    };
+
+    Card.prototype = {
+        removeCard: function() {
+            this.$element.remove();
+        }
+    }
+
+    var board = {
+        name: "Kanban Board",
+        addColumn: function(column) {
+            this.$element.append(column.$element);
+            initSortable();
+        },
+        $element: $("#board .column-container")
+    };
+
+    function initSortable() {
```

Comment:
can you move a card to empty column?

Reply:
no and I can't find the reason why... is bootstrap causing this?